src/operator/controllers/kafkaacls/intentsAdmin.go

```diff
@@ -76,7 +75,7 @@ func getTLSConfig(tlsSource otterizev1alpha2.TLSSource) (*tls.Config, error) {
 	}
 
 	pool := x509.NewCertPool()
-	rootCAPEM, err := ioutil.ReadFile(tlsSource.RootCAFile)
+	rootCAPEM, err := os.ReadFile(tlsSource.RootCAFile)
 	if err != nil {
```

Comment:
The package `ioutil` is deprecated, replaced by os which according to documentation has the same implementation